airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -41,6 +41,11 @@
       "customer_id": {
         "type": "string",
         "description": "Customer id must be specified  as a 10-digit number without dashes. More instruction on how to find this value in our <a href=\"https://docs.airbyte.io/integrations/sources/google-adwords#setup-guide\">docs</a>"
+      },
+      "conversion_window_days": {
+        "type": "string",
```

Comment:
adding `title` property to each of these parameters will make them display nicely in the ui. For example

```
"conversion_window_days": {
  "title": "Conversion Window"
   ...
} 
```
will display `Conversion Window in Days` in the UI instead of `conversion_window_days`